geoh5py/shared/utils.py

```diff
@@ -913,3 +919,9 @@ def remove_duplicates_in_list(input_list: list) -> list:
     :return: The sorted list
     """
     return sorted(set(input_list), key=input_list.index)
+
+
+def str2path(value):
+    if isinstance(value, str) and Path(value).exists():
```

Comment:
@domfournier, does the Path has to exist on disk? could it be an output path?

I do not know if there is a way to test if path is valid while not requesting for it to exists